profiles/learner-profiles.md

```diff
@@ -1,5 +1,15 @@
 ---
-title: FIXME
+title: Learner profiles
 ---
 
-This is a placeholder file. Please add content here.
+**Ana Fátima Fernandes dos Anjos Ferreira** is a biologist working on the effect of chemicals in aquatic environments. She works most of the time with R. R was never part of her training in the biology school, but she compensated for that with self-study, tutorials, both written and in Youtube. She did this self-learning mostly during weekends and free-time.  
+
+Eventually she became the acclaimed R authority in her lab. Since then, she has developed several scripts for her colleagues. This is a double-edged sword, because sometimes her colleagues underestimate the effort this takes. Nor does her supervisor fully appreciate the work it takes to write reusable R code. She finds it hard to get support; she has some friends she contacts with questions about R, but there is no one in her direct professional environment who can take the time to work deeply through an issue and spend time on her code to improve it.
+
+Often, she sees herself struggling with coding problems. That’s ok, just life, but she has this annoying feeling... this voice in the back of her head that says: “There should be an easier way for doing this...”. As a postdoc she is incredibly busy, and finds it hard to find time to get back to improving her old projects, although she knows she probably should.
+
+**Mzkizi Johnson** is an environmental scientist who is used to working with Python. He’s getting involved in a new project that has a lot of R and he needs to learn the ins and outs of it (best practices, etc.).
+
+Mzkizi was born in South Africa but has been living in Germany ever since his under-graduates. He likes cricket and has to do with watching streams, since the level of cricket competition is abismal in Germany.
```

Comment:
```suggestion
Mzkizi was born in South Africa but has been living in Germany ever since his undergraduate studies.
He likes cricket and has to make do with watching streams, since the level of cricket competition is abysmal in Germany.
```